crm_request/views/request.xml

```diff
@@ -18,6 +18,10 @@
                 <attribute name="string">Description</attribute>
             </xpath>
 
+            <field name="team_id" position="after">
+                <field name="alias_id" groups="base.group_no_one"/>
```

Comment:
Why adding group in a specific field? I would just let the field security be the same as the whole view no?

Reply:
I find that is a technical feature that pollute the view for the user.